packages/botonic-react/src/webchat/components/persistent-menu.jsx

```diff
@@ -12,7 +12,7 @@ const ButtonsContainer = styled.div`
   text-align: center;
 `
 
-export const OpenedPersistentMenu = ({ onClick, options }) => {
+export const OpenedPersistentMenu = ({ onClick, options, borderRadius }) => {
```

Comment:
should I change `{ onClick, options, borderRadius }` to `props`?

Reply:
I prefer the long form here, so it's clear what props are accepted by this component

Reply:
In my opinion it's preferably this way. You avoid the first step of... `const { borderRadius } = this.props`.